homeassistant/components/netatmo/__init__.py

```diff
@@ -105,6 +106,25 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     await data_handler.async_setup()
     hass.data[DOMAIN][entry.entry_id][DATA_HANDLER] = data_handler
 
+    async def handle_event(event):
+        """Handle webhook events."""
+        if event["data"]["push_type"] == "webhook_activation":
+            hass.async_create_task(
+                hass.config_entries.async_forward_entry_setup(entry, "light")
+            )
+
+    async def unload_light_platform():
```

Comment:
Why extract this as a function, it's only called once ?

Reply:
Let's just agree it shouldn't.